### PR TITLE
CORE-33793 Rename idSyncsEnabled to idSyncEnabled

### PR DESCRIPTION
**Pull request by wgamboa**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All tests pass and I've made any necessary test changes.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: src/components/Identity/createIdSyncs.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const setControlObject = (controlObject, cookieJar) => {
 };
 
 const createProcessor = (config, logger, cookieJar) => destinations => {
-  if (!config.idSyncsEnabled) {
+  if (!config.idSyncEnabled) {
     return Promise.reject(new Error("ID syncs not enabled"));
```
Comment (Aaronius) [inline src/components/Identity/createIdSyncs.js]: You might want to change this error message to "ID sync not enabled" or "ID synchronization not enabled".

File: src/components/Identity/index.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ const createIdentity = ({ config, logger, cookieJar }) => {
         return optIn.whenOptedIn().then(() => {
           if (
             !alreadyQueriedForIdSyncs &&
```
Comment (Aaronius) [inline src/components/Identity/index.js]: Do you want to change some of these variable names, or does it make sense to leave them as "syncs"?

File: src/components/Identity/createIdSyncs.js (hunk 1 above)
Comment (wgamboa) [reply src/components/Identity/createIdSyncs.js]: Thanks, I'll use the second one.

File: src/components/Identity/index.js (hunk 2 above)
Comment (wgamboa) [reply src/components/Identity/index.js]: I think it makes sense to leave them.

**Pull request merged**